client/gatsby-config.js

```diff
@@ -131,6 +132,12 @@ module.exports = {
         icon: 'src/assets/images/square_puck.png'
       }
     },
-    'gatsby-plugin-remove-serviceworker'
+    'gatsby-plugin-remove-serviceworker',
+    {
+      resolve: 'gatsby-plugin-robots-txt',
+      options: {
+        policy: [{ userAgent: '*', disallow: '/learn', crawlDelay: 2 }]
+      }
+    }
```

Comment:
```suggestion
```
I think we should be either using `robots.txt` or adding `noindex`, not both.  Looking [here](https://support.google.com/webmasters/answer/93710)  Google points out that the `robots.txt` will mean that the bot never goes into `/learn` and so never sees `noindex`.

Reply:
That is interesting. I did not know that. Thanks for sharing this. At another similar post from Google, I got the impression that perhaps both things needed to be done. Perhaps that was poor documentation on their part - never mind. Sure, let's go ahead with the `noindex` directive.